examples/starter-policy/fastapi/README.md

```diff
@@ -0,0 +1,230 @@
+# Setting up the Starter Policy
+
+This application assumes the following Polar policy is active:
+
+``` sh
```

Comment:
We use ruby for highlighting purposes; does that apply here? or maybe we can do polar now since we have a grammar?

Reply:
i changed this to use `polar` syntax highlighting, which looks to be rendering just fine in github :)